tests/system/providers/amazon/aws/example_boto3.py

```diff
@@ -0,0 +1,138 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow.decorators import task
+from airflow.models.dag import DAG
+from airflow.providers.amazon.aws.operators.boto3 import Boto3Operator
+from airflow.providers.amazon.aws.sensors.boto3 import Boto3Sensor
+
+
+AWS_CONNECTION_ID = "aws_default_conn"
+
+
+@task
+def generate_db_instance_identifier(**kwargs):
+    return f"example-database-{datetime.now():%Y-%m-%d-%H-%M-%S%z}"
+
+
+with DAG(
+    dag_id="example-boto3-dag",
```

Comment:
By convention AWS set the dag ID as the filename in system tests
```suggestion
    dag_id="example_boto3",
```